### PR TITLE
Using curses.ascii.isascii to check if char is ascii

### PR DESCRIPTION
**Pull request by IuryAlves**

### CODE REVIEW

File: tests/fixtures_regress/tests.py [hunk 1]
```diff
@@ -37,7 +38,7 @@
 
 
 def is_ascii(s):
-    return all(ord(c) < 128 for c in s)
+    return all(isascii(c) for c in s)
```
Comment (jarekwg) [inline tests/fixtures_regress/tests.py]: Could do all(map(isascii, s))

Comment (aaugustin) [inline tests/fixtures_regress/tests.py]: But that would be less pythonic :-)

Comment (jarekwg) [inline tests/fixtures_regress/tests.py]: I just find `map` sexy, totally worth the loss in clarity. :P

**Comment by timgraham** [comment]
buildbot, add to whitelist.

**Comment by timgraham** [comment]
After noticing the test failure, I found this: "While curses is most widely used in the Unix environment, versions are available for Windows, DOS, and possibly other systems as well."

I think one usage isn't worth an extra step for Windows developers trying to contribute to Django, but thanks for the suggestion!

**Comment by IuryAlves** [comment]
Sorry, i didn't realize that.

**Pull request closed without merging**